bpf/lib/lb.h

```diff
@@ -396,8 +402,15 @@ static __always_inline int reverse_map_l4_port(struct __ctx_buff *ctx, __u8 next
 				return ret;
 
 			if (port != old_port) {
+#ifdef ENABLE_SCTP
+				/* This will change the SCTP checksum, which we cannot fix right now.
+				 * This will likely need kernel changes before we can drop this.
```

Comment:
```suggestion
				 * This will likely need kernel changes before we can remove this.
```
Did you mean remove this code? The word "drop" is confusing in this context.